training-maquetado/scss/_texts.scss

```diff
@@ -12,6 +12,12 @@
 
 .text-4 {
   color: white;
-  font-size: $text-big;
+  font-size: $text-medium;
   text-transform: uppercase;
 }
+
+@media (min-width: $min-width-responsive) {
```

Comment:
when adding responsive do not change the current styles to theresponsive ones and add a new media query with the previous ones. Directly set the responsive styles inside the media query with max-width

Reply:
I'm not sure if I understand this. But, my idea doing it this way, was to simulate a "mobile first" experience. So if I delete all media queries it'll look like a mobile app  (as suggested in the Grid Video). I could change it if needed :)